spek-runner/junit5/src/main/kotlin/org/spekframework/spek2/junit/SpekTestEngine.kt

```diff
@@ -48,7 +48,9 @@ class SpekTestEngine : TestEngine {
             .map { it.toString() }
 
         val classSelectors = discoveryRequest.getSelectorsByType(ClassSelector::class.java)
-            .map {
+            .filter {
+                it.javaClass.isLocalClass.not()
```

Comment:
nit: `!it.javaClass.isLocalClass`

Reply:
maybe add a check for anonymous classes as well? Not sure if `local` == `anonymous`.

Reply:
Thanks! And I think adding a check for anonymous classes is also required so I will add it!